src/Storage/Storage.Management/help/Get-AzDataLakeGen2Item.md

```diff
@@ -24,41 +24,41 @@ This cmdlet only works if Hierarchical Namespace is enabled for the Storage acco
 ## EXAMPLES
 
 ### Example 1: Get a directory from a Filesystem, and show the details
-```
-PS C:\> $dir1 = Get-AzDataLakeGen2Item -FileSystem "filesystem1" -Path "dir1/"
-PS C:\> $dir1
```

Comment:
We should separate the command and output , like https://github.com/Azure/azure-powershell/blob/0b8fbc713e042cc33315a6f2049a34a4af33e495/src/Storage/Storage.Management/help/Enable-AzStorageContainerDeleteRetentionPolicy.md#example-1-enable-delete-retention-policy-for-blob-containers

As the sample has several command with several output, not sure how it should be handled.